Configuration/TypoScript/ContentElement/Shortcut.typoscript

```diff
@@ -5,31 +5,33 @@ tt_content.shortcut {
             fields {
                 shortcut = COA
                 shortcut {
-                    stdWrap.innerWrap = [|]
-                    10 = RECORDS
-                    10 {
-                        source.field = records
-                        tables = {$styles.content.shortcut.tables}
-                        conf {
-                            {$styles.content.shortcut.tables}.stdWrap.wrap = |###BREAK###
-                        }
-                    }
-                    stdWrap.split {
-                        token = ###BREAK###
-                        cObjNum = 1 |*|2|*| 3
-                        1 {
-                            current = 1
-                            stdWrap.wrap = |
+                    stdWrap {
```

Comment:
this change probably breaks functional test of shortcut element

Reply:
yes, just pushed the fix